include/k52/optimization/hooke_jeeves_method.h

```diff
@@ -0,0 +1,76 @@
+#ifndef HOOKE_JEEVES_METHOD_H_
+#define HOOKE_JEEVES_METHOD_H_
+
+#include <boost/shared_ptr.hpp>
+
+#include <k52/optimization/continuous_optimizer.h>
+
+namespace k52
+{
+namespace optimization
+{
+
+/// @class HookeJeevesMethod
```

Comment:
Add link to the source of your algorithm
It hugely differs from the one described in [WiKi](https://ru.wikipedia.org/wiki/%D0%9C%D0%B5%D1%82%D0%BE%D0%B4_%D0%A5%D1%83%D0%BA%D0%B0_%E2%80%94_%D0%94%D0%B6%D0%B8%D0%B2%D1%81%D0%B0)